python/ray/tune/trainable/trainable.py

```diff
@@ -31,7 +28,7 @@
     StorageContext,
     _exists_at_fs_path,
 )
-from ray.train._checkpoint import Checkpoint as NewCheckpoint
+from ray.train import Checkpoint
```

Comment:
For this class:

Before converting `NewCheckpoint` to `Checkpoint`, first:
* Convert all `Checkpoint` to `air.Checkpoint as LegacyCheckpoint`
* Then, convert all `NewCheckpoint` to `Checkpoint`